### DISCUSSION TITLE
不是啊？我走格子呢，这给我干哪来了？怎么回去啊？

### DISCUSSION THREAD

**Discussion by RETRN1** [share121/inter-knot#1597] [其他]
![freecompress-HnVideoEditor_2024_08_12_101815389](https://github.com/user-attachments/assets/5915fca6-5450-41eb-aff4-52796fc14900)
今天打委托时走这来了，谁能告诉我怎么回去啊！

**Comment by match-fire** [comment]
发gif不等待上传完，抬走下一个！

**Comment by ZhuYuan-148** [comment]
向治安局发送你的空洞入口位置。我会即刻出勤

**Comment by qingtaixu** [comment]
这个委托我接了

**Comment by qingtaixu** [comment]
这个委托我接了！

**Comment by IIID123** [comment]
这个委托我接了！

**Comment by Philia093-T** [comment]
你们都是什么人啊，接这么快
